R/read_alevin.R

```diff
@@ -39,7 +40,8 @@
 read_alevin <- function(quant_dir,
                         intron_mode = FALSE,
                         usa_mode = FALSE,
-                        which_counts = c("spliced", "unspliced")){
+                        which_counts = c("spliced", "unspliced"),
+                        version_10x = NULL){
```

Comment:
I know everything has been 10X so far, but let's make this variable name more generic? Maybe something like `kit_version`, or `tech_version`? The values we are using do encode that it is 10X, so I don't think it should be ambiguous at all.

Reply:
That makes sense. I updated it to be `tech_version`. Do we want the name in the table to also be `Tech version` or would we prefer something like `Technology`, `Technology version`, or`Technology used`?